client/src/components/AddNewProfile.js

```diff
@@ -127,10 +157,35 @@ const AddNewProfile = () => {
           onChange={handleChange}
         />
       </Form.Field>
+      <Form.Field>
+        <Dropdown
```

Comment:
All the other input fields have a label, so let's add a `<label>` here for visual consistency

Reply:
thanks for the feedback Coung :) 

I've made the changes, can you please take a look at them?